recipes/computer_vision/object_detection/model_server/Containerfile

```diff
@@ -1,8 +1,9 @@
 FROM registry.access.redhat.com/ubi9/python-311:1-52
+ARG PORT
 WORKDIR /locallm
 COPY requirements.txt /locallm/requirements.txt
 RUN pip install --upgrade pip && \
     pip install --no-cache-dir --upgrade -r requirements.txt
-COPY object_detection_server.py object_detection_server.py
-EXPOSE 8000
+COPY ../src/object_detection_server.py object_detection_server.py
```

Comment:
This is illeagle I believe.  You can not refer to a file outside of the context directory.